cli/flox/doc/manifest.toml.md

```diff
@@ -59,8 +59,23 @@ pkg-path = "python310Packages.pip"
 Flox will use the first format by default when automatically editing
 the manifest.
 
-```{.include}
-./include/package-names.md
+### Package names
```

Comment:
question nonblocking: should we just remove `### Package names` from the include file?

Reply:
No, the intention is that it gets included in a few different man pages with that header (just like how the options are included that way).